packages/tools/kadena-cli/README.md

```diff
@@ -161,73 +168,71 @@ kadena networks delete --network="mainnet" --network-delete="yes"
 
 ---
 
-## kadena keys
+## kadena wallets
 
-Tool to generate and manage keys
+Tool to generate and manage wallets
 
-| **Parameter**          | **Description**                                | **Required** | **Default value** |
-| ---------------------- | ---------------------------------------------- | ------------ | ----------------- |
-| create-wallet          | List all available networks                    | No           |                   |
-| import-wallet          | Import ( restore ) wallet from mnemonic phrase | No           |                   |
-| gen-hd                 | Generate key pair(s) from your wallet          | No           |                   |
-| gen-plain              | Generate plain key pair(s)                     | No           |                   |
-| change-wallet-password | Update the password for your wallet            | No           |                   |
-| delete-wallet          | Delete existing wallet from local env          | No           |                   |
-| decrypt                | Decrypt encrypted messsage                     | No           |                   |
-| list                   | List available key(s)                          | No           |                   |
+| **Parameter**   | **Description**                                     | **Required** | **Default value** |
+| --------------- | --------------------------------------------------- | ------------ | ----------------- |
+| add             | Add a new local wallet                              | No           |                   |
+| import          | Import ( restore ) wallet from mnemonic phrase      | No           |                   |
+| generate-keys   | Generate public/secret key pair(s) from your wallet | No           |                   |
+| change-password | Update the password for your wallet                 | No           |                   |
+| delete          | Delete existing wallet from local filesystem        | No           |                   |
+| list            | List wallets(s)                                     | No           |                   |
```

Comment:
`SubCommand` makes more sense Instead of `Parameter` 

imo `Required` and `Default Value` column not needed for SubCommand/Parameter. it's required only for `args/options`. So it could be removed from here and added into `Arguments & Options`. section